src/frontend/magnify/packages/components/src/i18n/TranslationProvider/TranslationsProvider.tsx

```diff
@@ -43,8 +44,8 @@ const LocaleContext = React.createContext<Maybe<LocaleContextInterface>>(undefin
 
 export const TranslationProvider = ({
   children,
-  locale = MagnifyLocales.EN,
-  defaultLocale = MagnifyLocales.EN,
+  locale = window.config.LANGUAGE_CODE ?? getDefaultLocale(),
+  defaultLocale = window.config.LANGUAGE_CODE ?? getDefaultLocale(),
```

Comment:
You could use `getDefaultLocale` directly. The test you are making here is also did within this method.